cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -1786,15 +1545,17 @@ private String insertNewWinnerRevision(SQLDatabase db,DocumentBody newWinner,
      *
      * @see #pickWinnerOfConflicts(long)
      */
-    private void setCurrent(SQLDatabase db, long sequence, boolean valueOfCurrent) {
+    public static void setCurrent(SQLDatabase db, long sequence, boolean valueOfCurrent) {
         ContentValues updateContent = new ContentValues();
         updateContent.put("current", valueOfCurrent ? 1 : 0);
         String[] whereArgs = new String[]{String.valueOf(sequence)};
         db.update("revs", updateContent, "sequence=?", whereArgs);
     }
 
-    private static DocumentRevision getFullRevisionFromCurrentCursor(Cursor cursor,
-                                                                          List<? extends Attachment> attachments) {
+    // TODO where should this be moved - DocumentRevision?
```

Comment:
There are a few of these statics hanging around, I think as part of the next pass we should list them all and see if we can break them into some reasonable groups that might better suggest where they belong (or perhaps allow us to place them appropriately in a hierarchies based on the callables that might need them (e.g. `abstract DocumentRevisionCallable` might be a possible way of containing to a specific group of functions).